pathml/datasets/pannuke.py

```diff
@@ -363,7 +367,7 @@ def train_dataloader(self):
         Yields (image, mask, tissue_type), or (image, mask, hv, tissue_type) for HoVer-Net
```

Comment:
I also saw that the train, validation and test function use this similar comment. The use of "Yields" when something is being returned as apposed to python's `yield` seems a bit confusing here.